packages/rainbowkit/src/wallets/latestWalletId.ts

```diff
@@ -0,0 +1,15 @@
+const storageKey = 'rk-latest-id';
+
```

Comment:
I was thinking of renaming this to something else since it's being used mostly in WalletButton API, but wanna keep the naming consistnent by always starting wtih `rk-`